pkgs/fc/default.nix

```diff
@@ -17,6 +17,10 @@ rec {
   collectdproxy = callPackage ./collectdproxy {};
   roundcube-chpasswd = callPackage ./roundcube-chpasswd {};
   fix-so-rpath = callPackage ./fix-so-rpath {};
+  ledtool = pkgs.writeScriptBin "fc-ledtool"
```

Comment:
Using `pkgs.writers.writePython3Bin` instead should be a bit cleaner here, it takes care of the interpreter shebang.

Reply:
TIL